src/js/components/Calculator.js

```diff
@@ -0,0 +1,79 @@
+import { MESSAGE } from "../utils/const.js";
+import { OPERATE, OPERATOR } from "../utils/math.js";
+import Components from "./Components.js";
+
+export default class Calculator extends Components {
+  putNumber(number) {
+    if (!this.isValidLength()) {
+      return alert(MESSAGE.LIMITED_NUMBER_WARNING_MESSAGE);
+    }
+
+    if (this.displayElement.textContent === "0") {
+      if (number === "0") return;
+    }
+    if (!this.existFirstNumber) {
+      this.setState({
+        ...this.state,
+        firstNumber: this.state.firstNumber + number,
+      });
+      this.updateDisplay();
+      return;
+    }
+
+    this.setState({
+      ...this.state,
+      secondNumber: this.state.secondNumber + number,
+    });
+    this.existSecondNumber = true;
+    this.updateDisplay();
+    return;
+  }
+
+  putOperator(operator) {
+    if (operator === "=") {
+      if (!this.operator) {
+        alert(MESSAGE.OPERATOR_WARNING_MESSAGE);
+        return;
+      }
+      if (!this.existSecondNumber) {
+        alert(MESSAGE.NEED_SECOND_NUMBER_MESSAGE);
+        return;
+      }
+      return this.calculate();
+    }
+
+    if (this.displayElement.textContent === "0")
+      this.setState({ ...this.state, firstNumber: "0" });
+    this.operator = operator;
+    this.existFirstNumber = true;
+
+    this.updateDisplay();
+  }
+
+  isValidLength() {
+    if (!this.existFirstNumber) {
+      if (this.state.firstNumber.length >= "3") {
+        return false;
+      }
+    }
+
+    if (this.state.secondNumber.length >= "3") {
+      return false;
+    }
+    return true;
```

Comment:
```suggestion
    return (this.state.secondNumber.length >= "3") ? false : true;
```
위의 뎁스도 줄여보면 좋을 것 같습니다! 이건 뭐 취향차이여서 ㅎㅎ